css-basics/css-basics_pseudo-park/index.html

```diff
@@ -7,12 +7,16 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <title>CSS Basics: Pseudo Park</title>
+    <link rel="stylesheet" href="./css/styles.css">
   </head>
   <body>
     <main class="park">
-      <div class="animal crocodile">🐊</div>
-      <div class="animal giraffe">🦒</div>
-      <div class="animal dino">🦖</div>
+      <div class="animal ">
```

Comment:
i think this div is not necessary as you could also call the classes of the animals combined with this like:
animal crocodile
animal giraffe
animal dino